sys/net/gnrc/network_layer/ipv6/gnrc_ipv6.c

```diff
@@ -690,6 +690,12 @@ static inline bool _pkt_not_for_me(kernel_pid_t *iface, ipv6_hdr_t *hdr)
     }
 }
 
+static inline bool _pkt_none_local_mcast(ipv6_hdr_t *hdr)
+{
+    return (hdr->dst->u8[0] == 0xff) &&
+           (hdr->dst->u8[1] >= IPV6_ADDR_MCAST_SCP_LINK_LOCAL);
```

Comment:
This ignores the [flags of a multicast address](https://tools.ietf.org/html/rfc4291#section-2.7). `ff12::1` would also be a legal link-local address.